examples/transformerconfigs/crd/README.md

```diff
@@ -0,0 +1,133 @@
+## Transformer Configurations - CRD
+
+This tutorial shows how to add transformer configurations to support a CRD type.
+
+### Get Default Config
+Get the default transformer configurations by
+
+<!-- @saveConfig @test -->
+```
+kustomize config save -d ~/.kustomize/config
+```
+The default configurations are save in directory `~/.kustomize/config` as several files
+
+> ```
+>  commonannotations.yaml  commonlabels.yaml  nameprefix.yaml  namereference.yaml  namespace.yaml  varreference.yaml
+> ```
+
+### Add Config for a CRD
+All transformers will be involved for a CRD type. The default configurations already include some common fieldSpec for all types:
+
+- nameprefix is added to `.metadata.name`
+- namespace is added to `.metadata.namespace`
+- labels is added to `.metadata.labels`
+- annotations is added to `.metadata.annotations`
+
+Thus those fieldSpec don't need to be added to support a CRD type.
+Consider a CRD type `MyKind` with fields
+- `.spec.secretRef.name` reference a Secret
+- `.spec.beeRef.name` reference an instance of CRD `Bee`
+- `.spec.containers.commands` as the list of container commands
+- `.spec.selectors` as the label selectors
+
+Add following file to configure the transformers for the above fields
+<!-- @addConfig @test -->
+```
+cat > ~/.kustomize/config/mykind.yaml << EOF
+
+commonLabels:
+- path: spec/selectors
+  create: true
```

Comment:
Hmm. It is called `create` in yaml.
I guess you probably have document explaining what this field means.
Otherwise, it may be confusing for the user.

Reply:
I'll add this explanation in `examples/transformerconfigs/README.md`